docs-src/concepts/what-is-a-default-data-converter.md

```diff
@@ -0,0 +1,27 @@
+---
+id: what-is-a-default-data-converter
+title: What is a default Data Converter?
+sidebar_label: Default Data Converter
+description: The default Data Converter is used by the Temporal SDK to convert objects into bytes using a series of Payload Converters.
+tags:
+  - term
+  - explanation
+---
+
+Each Temporal SDK includes and uses a default Data Converter.
+The default Data Converter converts objects into bytes using a series of Payload Converters.
+In most SDKs, the default converter supports binary, JSON, and Protobufs and encodes values in the following order:
+
+- Null
+- Binary
```

Comment:
I would add that it's "for Protobuf objects that can't be converted into jsons". It's really rare when users may face this one.